maker-core/delegates.md

```diff
@@ -1,5 +1,39 @@
-# TBC
+# Constitutional Delegates (CDs)
 
-{% hint style="info" %}
-Sorry! We haven't been able to publish documentation on this topic yet. Please check back soon.
-{% endhint %}
+## Overview
+Delegates are ecosystem actors who operate through Protocol Delegation System smart contracts. MKR holders can delegate their voting power to them without the need to hand over the custody of their MKR.
+
+Constitutional Delegates (CDs) are one type of [Constitutional Conservers](https://endgame.makerdao.com/maker-core/tbc-governance/constitutional-conservers). They receive various benefits and are subject to specific requirements. They must put in place the governance strategies defined by the CVCs.
+
+CDs must not engage with other ecosystem actors or in operational activities. They are not allowed to compensate MKR holders that delegate to them. They must aim to provide governance information and research material to MKR holders.
+
+## Purpose
+
+The Constitutional Delegate role spares Delegates from having to create governance strategies. CDs must follow strategies that have been designed by CVC members.
+CDs do not take part in the creation of the strategies, they execute them. The risk of Delegates following their own agenda, since they are not required to hold any MKR, is then  eliminated.
+
+## Income
+CDs can receive compensation from the Maker Protocol. The compensation goes to the top 14 CDs that have the highest amount of combined MKR delegated to their delegate contracts and fulfill the participation requirements.
+
+There are two income levels for the top 14 qualified CDs: Prime and Reserve.  The 7 top qualified CDs are called the Prime Constitutional Delegates (PCDs). The top 8 to 14 qualified CDs are called the Reserve Constitutional Delegates (RCDs).
+
+Each income level has a monthly total compensation.
```

Comment:
A total monthly compensation of xxx MKR/DAI is distributed equally among eligble Prime CDs. Similarly, a total monthly compensation of yyy MKR/DAI is distributed equally among eligible Reserve CDs.

Reply:
I thought we agreed on leaving specific numbers out since they are subject to changes?

Reply:
@LongForWisdom, any thoughts?